js/viz/chart.d.ts

```diff
@@ -74,6 +74,7 @@ import {
     WordWrap,
     ZoomPanAction,
     ChartsColor,
+    ValueAxisVisualRangeUpdateMode,
```

Comment:
Seems like we should also reexport it here

Reply:
fixed